zanshinsdk/client.py

```diff
@@ -967,6 +967,20 @@ def get_organization_scan_target_group(self, organization_id: Union[UUID, str],
                              f"/organizations/{validate_uuid(organization_id)}/scantargetgroups/"
                              f"{validate_uuid(scan_target_group_id)}").json()
 
+    def iter_compartments_from_group(self, organization_id: Union[UUID, str],
```

Comment:
I think the method should be named `iter_scan_target_group_compartments` to follow the same standard as the rest of the SDK.